reference/conanfile/methods.rst

```diff
@@ -720,6 +724,18 @@ instance of the package manager. It validates whether your current settings are
 cross-building, and if so, it will update the package name to be installed according to
 ``self.settings.arch``.
 
+To install more than one package at once:
+
+..  code-block:: python
+
+            packages = [("vim", "nano", "emacs"), "firefox", "chromium"]
```

Comment:
Please add the ``def system_requirements(self):`` in the code block for better understanding (and fix indentation if needed)

Reply:
Done